ui/packages/catalog-ui-search/src/main/webapp/component/content/content.js

```diff
@@ -110,7 +110,11 @@ module.exports = Backbone.AssociatedModel.extend({
   },
   turnOnDrawing(model) {
     this.set('drawing', true)
+    model.set('drawing', true)
     this.set('drawingModel', model)
+    if (model.get('mode') === 'line' && !model.get('lineWidth')) {
+      model.set('lineWidth', 1)
+    }
```

Comment:
Do we want to do the same thing here as we did with `turnOffDrawing` and move the setting of attributes on the model to location-old to be consistent?